src/lv_api_map_v9_1.h

```diff
@@ -87,10 +87,16 @@ extern "C" {
 #define LV_LABEL_LONG_SCROLL_CIRCULAR  LV_LABEL_LONG_MODE_SCROLL_CIRCULAR
 #define LV_LABEL_LONG_CLIP             LV_LABEL_LONG_MODE_CLIP
 
+#define lv_anim_set_playback_delay lv_anim_set_reverse_delay
+#define lv_anim_set_playback_duration lv_anim_set_reverse_duration
+
 /**********************
  * DEPRECATED FUNCTIONS
  **********************/
 
+#define lv_anim_set_time lv_anim_set_duration
```

Comment:
So we're deprecating this?

I see these are still there.

```c
/**
 * Legacy `lv_anim_set_time` API will be removed soon, use `lv_anim_set_duration` instead.
 */
void lv_anim_set_time(lv_anim_t * a, uint32_t duration);
```

```c
void lv_anim_set_time(lv_anim_t * a, uint32_t duration)
{
    lv_anim_set_duration(a, duration);
}
```

Should they be removed in this PR or at a later time?

Reply:
> Should they be removed in this PR or at a later time?

cc:  @kisvegabor 

I think we will need Gábor's input on this I'm not aware of "the plan" for it.  Forgive me if you are already keenly aware of this, but here is the history of it in case it is helpful (from what I can tell from the repository history):

- 18-Nov-2023  c526653   Gábor changed the `lv_anim_t` member name from `time` => `duration`.
- 01-Aug-2024  1d14386  Liam added `lv_anim_set_duration()` and `lv_anim_set_time()` functions (the latter calling the former).  This appears related the changed field name and perhaps preventing code in the field from breaking(?).

I don't know what the plan is about the `lv_anim_set_time()`, but if the presence of `lv_api_map_v9_1.h` enables us to remove it, I would condone doing so and would be happy to insert that into this PR if desired.

Awaiting feedback from the two of you,
Vic

Reply:
In https://github.com/lvgl/lvgl/commit/1d14386b99cdff731cc3367d1d8b5e96ffb8fcea I only changed it from `static inline` function in the header to a normal function in the C file ;)

```diff
-static inline void lv_anim_set_time(lv_anim_t * a, uint32_t duration)
-{
-    lv_anim_set_duration(a, duration);
-}
+void lv_anim_set_time(lv_anim_t * a, uint32_t duration);
```

Reply:
> In [1d14386](https://github.com/lvgl/lvgl/commit/1d14386b99cdff731cc3367d1d8b5e96ffb8fcea) I only changed it from a `static inline` function

Ah!  Got it.  I misinterpreted what I saw.

Reply:
I confirm that `lv_anim_set_time` is deprecated in favor of `lv_anim_set_duration`.

`lv_anim_set_time` should be added only to `lv_api_map_v9_1.h`, and all references should be updated to `lv_anim_set_duration`.

Reply:
> `lv_anim_set_time` should be added only to `lv_api_map_v9_1.h`

Done.

> and all references should be updated to lv_anim_set_duration

Done (in most recent commit).

So what I am hearing (correct me if I'm wrong) is that it's not time to delete `lv_anim_set_time()` yet.  Correct?

cc:  @liamHowatt

Reply:
> I interpret that as it is time to remove the functions. I can't see an advantage to the API map entry existing at the same time as the function.

This makes sense to me, if only not to have something in the future we have to remember, although keeping a list of such things (so it does get remembered) is very easy.  @kisvegabor What do you think?